scripts/import/import_EPMC.pl

```diff
@@ -137,9 +137,16 @@ sub import_citations{
     my $data           = shift;
     my $type           = shift;
 
+    open (my $error_log, ">>$species\_$type\_notfound\_" . log_time() . "\.log") or die "Failed to open log file: $!\n";
```

Comment:
It would be easier to understand the file purpose if we replace 'notfound' with 'errors' in the file name